maintenance_stock/models/maintenance_equipment.py

```diff
@@ -0,0 +1,59 @@
+# © 2020 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import api, fields, models
+
+
+class MaintenanceEquipment(models.Model):
+    _inherit = "maintenance.equipment"
+
+    allow_consumptions = fields.Boolean(
+        groups="stock.group_stock_user",
+    )
+    default_consumption_warehouse_id = fields.Many2one(
+        string="Default Consumption Warehouse",
+        comodel_name="stock.warehouse",
+        groups="stock.group_stock_user",
+    )
+
+    @api.onchange("allow_consumptions")
+    def _onchange_allow_consumptions(self):
+        if not self.allow_consumptions:
+            self.default_consumption_warehouse_id = False
+
+    def action_view_stock_picking_ids(self):
+        self.ensure_one()
+        action = self.env.ref("stock.action_picking_tree_all").read()[0]
```

Comment:
Starting from v14, `read()[0]` calls show be replaced by `_for_xml_id()` because any user with lower access rights than admin should fire an error, see

https://github.com/OCA/maintenance/blob/9e2477805e625b01e967a98d1a1fbe1d9c245034/maintenance_timesheet/models/maintenance_request.py#L71-L73